### PR TITLE
Override missed equals hashcode serialVersionUID

### PR DESCRIPTION
**Pull request by putrenkov**
**A Brief Overview**
In some classes missed equals hashcode serialVersionUID the implements Serializable

**Link to QA issue**
QA-3182 - https://github.com/BroadleafCommerce/QA/issues/3182

### CODE REVIEW

**Comment by charchar** [comment]
@putrenkov 	Looks like the class import ordering is incorrect. Assuming you are using intellij, please import these settings to your IDE: https://github.com/BroadleafCommerce/CodeStyle/blob/master/src/main/resources/broadleaf-java.importorder. FYI the order should be `org`, `com`, `java`, `javax`.

**Comment by putrenkov** [comment]
@charchar Thanks I will fix it

**Comment by charchar** [comment]
@putrenkov After talking with @danielcolgrove we want to fix the scenarios where a `Serializable` class has one of `equals()` or `hashCode()` but not the other. I pushed a test to `integration` module that checks for this situation and fails on a build. Can you resolve the 4 failing classes? You should be able to see them by running the test `EqualsHashCodeMethodsTest` in your IDE. Thanks!

**Comment by putrenkov** [comment]
@charchar Hi, I ran a test it was successful.

File: admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/url/domain/URLHandlerImpl.java [hunk 1]
```diff
@@ -45,6 +45,9 @@
 import javax.persistence.InheritanceType;
 import javax.persistence.Table;
 
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
```
Comment (charchar) [inline admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/url/domain/URLHandlerImpl.java]: @putrenkov Still seeing import ordering issues throughout PR.

**Review by charchar — CHANGES_REQUESTED** [review]

File: admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/url/domain/URLHandlerImpl.java (hunk 1 above)
Comment (putrenkov) [reply admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/url/domain/URLHandlerImpl.java]: @charchar Hi, could you check, is it better?

File: admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/rulebuilder/dto/DataWrapper.java [hunk 2]
```diff
@@ -27,6 +27,8 @@
 import java.io.Serializable;
 import java.util.ArrayList;
 
+import org.apache.commons.lang3.builder.HashCodeBuilder;
```
Comment (charchar) [inline admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/web/rulebuilder/dto/DataWrapper.java]: @putrenkov This is out of order

**Review by charchar — CHANGES_REQUESTED** [review]

**Review by nathandmoore — CHANGES_REQUESTED** [review]
Need to change `obj.getClass() != getClass()` to `getClass().isAssignableFrom(obj.getClass)` in each `equals()`.

**Pull request merged**